src/details-element-polyfill/polyfill.coffee.erb

```diff
@@ -91,14 +117,34 @@ clickEventIsSignificant = (event) ->
     event.target.isContentEditable
   )
 
-onTogglingClick = (callback) ->
+keydownEventIsSignificant = (event) ->
+  not (
+    event.defaultPrevented or
+    (event.keyCode != 32 and event.keyCode != 13) or
+    event.altKey or
+    event.ctrlKey or
+    event.metaKey or
+    event.shiftKey or
+    event.target.isContentEditable
+  )
```

Comment:
Since this is so similar to `clickEventIsSignificant`,  maybe there should be a single `eventIsSignificant` function with the common parts. And the event handlers can check the unique parts, e.g.:
```coffeescript
addEventListener "keydown", (event) ->
  if eventIsSignificant(event)
    if event.keyCode in [13, 32]
      ...
```